eZ/Publish/Core/FieldType/Price/SearchField.php

```diff
@@ -8,79 +8,11 @@
  */
 namespace eZ\Publish\Core\FieldType\Price;
 
-use eZ\Publish\SPI\Persistence\Content\Field;
-use eZ\Publish\SPI\Persistence\Content\Type\FieldDefinition;
-use eZ\Publish\SPI\FieldType\Indexable;
-use eZ\Publish\SPI\Search;
+use eZ\Publish\Core\FieldType\Unindexed;
 
 /**
  * Indexable definition for string field type.
  */
-class SearchField implements Indexable
+class SearchField extends Unindexed
```

Comment:
No need for this, you should instead remove this file completely and instead register `Unindexed` in the container, as is [already the case for some other field types](https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/Core/settings/indexable_fieldtypes.yml#L148).